sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -324,7 +324,15 @@ private[ui] class ExecutionPagedTable(
 
             <th>
               <a href={headerLink}>
+                {if (header == "Duration") {
+                <span data-toggle="tooltip" data-placement="top"
+                      title="Time from query submission to completion
+                   (or if still executing, time since submission)">
```

Comment:
Indent this more

Reply:
@srowen, review comments fixed, please check.